spec/lucky_web/input_helpers_spec.cr

```diff
@@ -36,6 +36,24 @@ describe LuckyWeb::InputHelpers do
     HTML
   end
 
+  it "renders checkbox inputs" do
+    view.checkbox(form.first_name, value: "1").to_s.should contain <<-HTML
+    <input type="checkbox" name="user:first_name" value="1"/>
+    HTML
+
+    view.checkbox(form.first_name, value: "1", class: "cool").to_s.should contain <<-HTML
+    <input type="checkbox" name="user:first_name" value="1" class="cool"/>
+    HTML
+
+    view.checkbox(form.first_name, value: "1").to_s.should contain <<-HTML
```

Comment:
I think extracting a method to explain this matcher might aid in readability. I'm thinking something like this:

```crystal
view.checkbox(form.first_name, value: "1").to_s.should have_unchecked_value("0")

def have_unchecked_value(value)
  contain <<-HTML
  <input type="hidden" name="user:first_name" value="#{value}" id=""/>
  HTML
end
```

What do you think?

Reply:
That makes total sense!